end-to-end-test/shared/specUtils.js

```diff
@@ -166,6 +166,11 @@ function reactSelectOption(parent, optionText) {
     return parent.$('.Select-option='+optionText);
 }
 
+function getNewReactSelectOptions(parent) {
```

Comment:
is there a more permanent/descriptive name for this? "New" is only accurate until it's not new anymore.

Reply:
I have renamed this to `getSelectCheckedOptions`